lib/api/fields/class-beans-fields.php

```diff
@@ -94,28 +117,29 @@ private function add() {
 
 		// Register fields.
 		self::$registered[ $this->context ][ $this->section ] = $fields;
-
 	}
 
 	/**
-	 * Standadrize field to beans format.
+	 * Standardize the field to the Beans' format.
```

Comment:
What is this "format" that is referred to? Where can I find out more about it? (rhetorical)